dali/test/python/test_operator_transpose.py

```diff
@@ -78,19 +81,25 @@ def iter_setup(self):
         data = self.iterator.next()
         self.feed_input(self.data, data, layout=self.layout)
 
-def check_transpose_vs_numpy(device, batch_size, shape, permutation):
-    eii1 = RandomDataIterator(batch_size, shape=shape)
-    eii2 = RandomDataIterator(batch_size, shape=shape)
-    compare_pipelines(TransposePipeline(device, batch_size, "HWC", iter(eii1), permutation=permutation),
-                      PythonOpPipeline(lambda x: transpose_func(x, permutation), batch_size, "HWC", iter(eii2)),
-                      batch_size=batch_size, N_iterations=10)
+def check_transpose_vs_numpy(device, batch_size, dim, total_volume, permutation):
+    max_shape = [int(math.pow(total_volume/batch_size, 1/dim))] * dim
+    print("Testing", device, "backend with batch of", batch_size, "max size", max_shape)
+    print("permutaiton ", permutation)
```

Comment:
```suggestion
    print("permutation ", permutation)
```
or even remove those prints if they were for debugging purposes